src/index.html

```diff
@@ -4,8 +4,123 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Audio Player</title>
+  <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h2>Audio Player</h2>
+  <div class="container">
+    <div class="playerContainer">
+      <div class="header">
+        <hr />
+      </div>
+      <div class="playerContent">
+        <div class="albumPicture">
+          <img class="image" src="./images/Gladiator.jpeg" >
+        </div>
+        <div class="trackInfo">
+          <div class="player">
+            <div class="nameContainer">
+              <div class="track">
+                <p>Scratching The Surface</p>
+              </div>
+              <div class="artist">
+                <p>Quincy Larson</p>
+              </div>
+            </div>
+            <div class="timeBar">
+              <div id=currentTime class="music-progress__time-item music-current-time start_track"></div>
+              <div id=duration class="music-progress__time-item music-duration-time end_track"></div>
+            </div>
+            <div class="progress_bar">
+              <input class="song" type="range" min='0 'max="100" >
+            </div>
+            <div class="panelPlayers">
+              <div class="left_button" onclick='selectingTrack(-1)'>
+                <img src="./images/previous.png" alt="">
+              </div>
+              <div class="playButton">
+                <img id=play class='play' src="./images/play.png" alt="" onclick='play(this)' />
+              </div>
+              <div class="pauseButton">
+                <img id=pause class='pause' src="./images/pause.png" alt="" onclick='pause(this)'/>
+              </div>
+              <div class="rightButton" onclick='selectingTrack(1)'>
```

Comment:
remove onclick functions which are not declared, for example "selectingTrack", "play(this)", "pause(this)"

Reply:
Fixed in 4-add-audio-player-functionality-by-using-js branch